src/helpers/figures/charts/runtime/chart_data_extractor.ts

```diff
@@ -402,10 +402,18 @@ function interpolateData(
         )[0];
       }
       case "logarithmic": {
+        const logLabels: number[] = [];
+        const filteredValues: number[] = [];
+        for (let i = 0; i < values.length; i++) {
+          if (normalizedLabels[i] > 0) {
+            logLabels.push(values[i]);
+            filteredValues.push(Math.log(normalizedLabels[i]));
+          }
+        }
```

Comment:
Just a small though : here we can check if we have some values in `filteredValues` or if it's empty, cause as it's empty we can simply return an empty array. Same comment can be applied to the line 398 :)